### PR TITLE
OIDC Azure AD

### LINKED ISSUE #60: Azure AD auth
Employee facing auth

### PR DESCRIPTION
**Pull request by amundlrohne**

### CODE REVIEW

File: apps/api/src/routes/UserRoutes.ts [hunk 1]
```diff
@@ -18,6 +20,20 @@ userRoutes.get('/', async (req, res, next) => {
     }
 });
 
+userRoutes.get('/me', ensureAuthentication, async (req, res, next) => {
```
Comment (perkynades) [inline apps/api/src/routes/UserRoutes.ts]: Rename route from "/me" to "/currentUser"?

File: apps/api/src/routes/middleware/errorHandler.ts [hunk 2]
```diff
@@ -11,9 +11,9 @@ export function errorHandler(error: Error, request: Request, response: Response,
...
-        const error = new BadRequestError({ message: ServerErrorMessage.invalidData()});
+        const error = new BadRequestError({ message: ServerErrorMessage.invalidData() });
         return response.status(error.httpStatus).json(error.toResponse());
     }
 
-    return response.status(StatusCodes.INTERNAL_SERVER_ERROR).json('Something went wrong.. Try again alter');
-}
\ No newline at end of file
+    return response.status(StatusCodes.INTERNAL_SERVER_ERROR).json('Something went wrong.. Try again later.' + error);
+}
```
Comment (perkynades) [inline apps/api/src/routes/middleware/errorHandler.ts]: Is this safe? Was just thinking its not a good idea to expose the error that cause the 502

File: apps/frontend/src/components/misc/breadcrumbs/Breadcrumbs.tsx [hunk 3]
```diff
@@ -1,33 +1,27 @@
...
 
 export const Breadcrumbs = (): ReactElement => {
     const location = useLocation();
 
-    const breadcrumbs = location.pathname
-        .split('/')
-        .filter((link) => link !== '');
+    const breadcrumbs = location.pathname.split('/').filter((link) => link !== '');
+
+    console.log(breadcrumbs);
```
Comment (perkynades) [inline apps/frontend/src/components/misc/breadcrumbs/Breadcrumbs.tsx]: remove log

**Review by perkynades — COMMENTED** [review]
Very good work sir, look at my comments

File: apps/api/src/routes/middleware/errorHandler.ts (hunk 2 above)
Comment (amundlrohne) [reply apps/api/src/routes/middleware/errorHandler.ts]: Woops, good catch. That is left over from debugging. However, maybe we should look into separate logging for the dev environment?

File: apps/api/src/routes/UserRoutes.ts (hunk 1 above)
Comment (amundlrohne) [reply apps/api/src/routes/UserRoutes.ts]: will do!

File: apps/frontend/src/components/misc/breadcrumbs/Breadcrumbs.tsx (hunk 3 above)
Comment (amundlrohne) [reply apps/frontend/src/components/misc/breadcrumbs/Breadcrumbs.tsx]: Will do!

**Review by perkynades — APPROVED** [review]

**Pull request merged**